infra/gcp/ensure-test-storage.sh

```diff
@@ -0,0 +1,27 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The service account name for the image promoter test account
+PROMOTER_TEST_SVC="k8s-infra-gcr-promoter-test"
+
+# GCP project names.
+STAGING_PROJECT="k8s-staging-cip-test"
+PROD_PROJECT="k8s-cip-test-prod"
+
+ALL_PROJECTS=("${TEST_PROJECT}")
+
+# Regions for test.
+TEST_REGIONS=(us eu asia)
+
+# GCR service account permissions
+PERMISSIONS=("objectAdmin" "admin")
+
+# Empower image promoter
+for prj in "${ALL_PROJECTS[@]}"; do
+    color 6 "Empowering image promoter to GCR: ${prj}"
+    for r in "${TEST_REGIONS[@]}"; do
```

Comment:
We want the test svc acct to have full permissions for STAGING_PROJECT, because that will mimic real-life usage. I don't think the empowering should be tied to regions.

Reply:
To clarify, the test svc account should only have regional perms for the k8s-cip-test-prod, not k8s-staging-cip-test, because this mimics real-life usage.

Reply:
Cool removing regions.